lib/sqlalchemy/testing/suite/test_reflection.py

```diff
@@ -1148,6 +1148,38 @@ def test_autoincrement_col(self):
             assert id_.get("autoincrement", True)
 
 
+class ComputedReflectionTest(fixtures.TablesTest):
```

Comment:
I've added a new class here, instead of modifying `ComponentReflectionTest` because a lot of tests needed to be changed if a new table was added. I think it is ok, since there will be other tests on the computed detection once it is complete